src/test/mochitest/head.js

```diff
@@ -598,13 +598,11 @@ function waitForLoadedSources(dbg) {
  * @static
  */
 function selectSource(dbg, url, line) {
-  info(`Selecting source: ${url}`);
```

Comment:
hmm these infos might be useful, why might we want to remove them?

Reply:
the new action logging is better :)